source/cpptooling/data/representation.d

```diff
@@ -1381,14 +1382,6 @@ const:
     }
 }
 
-/// Dictates how the namespaces are merged.
-enum MergeMode {
-    /// Merge everything except nested namespaces.
-    shallow,
-    /// Merge everything.
-    full
-}
-
 @safe struct CppNamespace {
```

Comment:
Seems like it automatically merges every namespace that already exists (first namespace) and ignores the rest?